model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -89,13 +89,8 @@ class LicenseInfoResolver(
         val filteredDetectedLicenseInfo =
             licenseInfo.detectedLicenseInfo.filterCopyrightGarbage(copyrightGarbageFindings)
 
-        val allCopyrights = filteredDetectedLicenseInfo.findings.flatMap { finding ->
-            finding.copyrights.map { it.statement }
-        }
-        val processedCopyrightStatements = CopyrightStatementsProcessor().process(allCopyrights).toMap()
         val unmatchedCopyrights = mutableMapOf<Provenance, MutableSet<CopyrightFinding>>()
-        val resolvedLocations =
-            resolveLocations(filteredDetectedLicenseInfo, processedCopyrightStatements, unmatchedCopyrights)
+        val resolvedLocations = resolveLocations(filteredDetectedLicenseInfo, unmatchedCopyrights)
```

Comment:
Could / should we easily create a test case for the case mentioned in the commit message?

Reply:
Done.

Reply:
Thank you!